src/protocol/relayer.md

```diff
@@ -0,0 +1,64 @@
+# Layer 1 Relayer/Bridge Protocol
+
+The Fuel relayer/bridge protocol is a set of rules that govern the interaction between the Fuel blockchain and the
+Layer 1 (L1) blockchain (e.g. Ethereum).  
+
+The Fuel blockchain can emit messages that will be processed by the smart contract on the L1 blockchain. The smart
+contract on the L1 can also emit events that will be processed by the Fuel blockchain.
+This is used to move any data between the L1 blockchain and the Fuel blockchain.
+
+## Fuel Message Outbox
+
+The message outbox is the set of messages sent to the L1 blockchain from the Fuel blockchain.
+
+## Fuel Event Inbox
+
+The event inbox is the set of events received from the L1 blockchain by the Fuel blockchain.
+
+The block producer will receive the list of events from the L1 by some relayer, and then include both the count and the
+merkle root of the events in the block header.
```

Comment:
It doesn't look like we actually put the event count into the header

Reply:
This count is implicitly derived from the da height and reconciling that with L1 data

Reply:
Oh. Whoops. We only include it for the outbox.